commit 31a2795228df2ccd08888b96c0bb7c34135dd42d
Author: graineeg <graineeg@gmail.com>
Date:   Fri Sep 8 22:00:38 2017 +0200

    wifi-tethering
    
    Change-Id: I5d075f24e2360db0da5d628fe3e06e9d02674e1a

diff --git a/server/SoftapController.cpp b/server/SoftapController.cpp
index 4a7c8d7..8461e43 100755
--- a/server/SoftapController.cpp
+++ b/server/SoftapController.cpp
@@ -53,7 +53,7 @@ using android::base::WriteStringToFile;
 #include "wpa_ctrl.h"
 #endif
 
-std:: string hostapd_unix_file(StringPrintf("/data/misc/wifi/hostapd/wlan0"));
+std:: string hostapd_unix_file(StringPrintf("/data/misc/wifi/hostapd/ap0"));
 #ifdef LIBWPA_CLIENT_EXISTS
 static const char HOSTAPD_DHCP_DIR[]    = "/data/misc/dhcp";
 #endif
@@ -289,7 +289,7 @@ int SoftapController::setSoftap(int argc, char *argv[]) {
             "hw_mode=%c\n"
             "ignore_broadcast_ssid=%d\n"
             "wowlan_triggers=any\n",
-            argv[2], argv[3], channel, (channel <= 14) ? 'g' : 'a', hidden));
+            /*argv[2]*/"ap0", argv[3], channel, (channel <= 14) ? 'g' : 'a', hidden));
 
     std::string fbuf;
     if (argc > 7) {
